bin/stress-test/src/store/mod.rs

```diff
@@ -288,6 +289,16 @@ async fn check_nullifiers_by_prefix(
     (start.elapsed(), response.into_inner())
 }
 
+// LOAD STATE
+// ================================================================================================
+
+pub async fn load_state(data_directory: &Path) {
+    let start = Instant::now();
+    let _state = State::load(data_directory).await.unwrap();
+    let elapsed = start.elapsed();
+    println!("State loaded in {elapsed:?}");
+}
```

Comment:
This works. Another approach would be start the store service and then wait until we get a response from the `status` endpoint - right? I'm assuming this would be more complicated though?